.github/workflows/codeforafrica-deploy-review-app.yml

```diff
@@ -88,15 +88,12 @@ jobs:
 
   destroy_review_app:
     runs-on: ubuntu-latest
-    # only run when a pull request is closed
-    if: github.event.action == 'closed'
+    if: github.event_name == 'pull_request' && github.event.action == 'closed'
     steps:
-      - name: Destroy the review app
+      - name: Destroy review app
         uses: dokku/github-action@master
         with:
-          # destroy a review app
           command: review-apps:destroy
-          git_remote_url: ${{ env.DOKKU_REMOTE_URL }}/${{ env.APP_NAME }}
-          # specify a name for the review app
+          git_remote_url: ${{ env.DOKKU_REMOTE_URL }}/codeforafrica-ui
```

Comment:
Have we tested this and it actually deletes the _review_ app (e.g. `codeforafrica-ui-789`) instead of the _dev_ app (`codeforafrica-ui`)?

Reply:
✅ ... saw that both URL and Name are [set](https://github.com/dokku/github-action/blob/master/example-workflows/review-app.yml#L40-L42).

Reply:
I have not tested this yet because running the workflow requires executing it in the main branch. Do you have any suggestions on how we can safely test this without affecting the main environment, or should we proceed with a cautious test in the main branch?

Reply:
There are tools @VinneyJ e.g. [act](https://github.com/nektos/act) but I'm not 100% convinced they're worth it... Code looks legit so I'm happy for us to proceed with deployment.